estate_account/models/estate_property.py

```diff
@@ -0,0 +1,38 @@
+from odoo import models, Command
+
+
+class InheritedEstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def action_sold(self):
+        invoices = []
+        for record in self:
+            journal = self.env['account.journal'].search([('type', '=', 'sale')])
```

Comment:
This could return several journal, add `limit=1` to make sure you only get one :+1: